docs/runtimes/configuring-job-resources.md

```diff
@@ -207,21 +208,20 @@ Preemption mode has three values:
 To change the default function preemption mode, it is required to override mlrun the api configuration 
 (and specifically "MLRUN_FUNCTION_DEFAULTS__PREENPTION_MODE" envvar to either one of the above modes).
 
-### UI configuration
-
-```{admonition} Note
-Relevant when MLRun is executed in the [Iguazio platform](https://www.iguazio.com/docs/latest-release/).
-```
-
-You can configure Spot node support when creating a job, rerunning an existing job, and creating an ML function. 
-The **Run on Spot nodes** drop-down list is in the **Resources** section of jobs. 
-Configure the Spot node support for individual Nuclio functions when creating a function in the **Configuration** tab, under **Resources**. 
-
 ### SDK configuration
 
 Configure preemption mode by adding the `with_preemption_mode` parameter in your Jupyter notebook, and specifying a mode from the list of values above. <br>
 This example illustrates a function that cannot be scheduled on preemptible nodes:
 
+
+```
+# Only run on non-spot instances
+fn.with_node_selection(node_selector={"app.iguazio.com/lifecycle" : "non-preemptible"})
```

Comment:
This node selector label only applies for iguazio. I believe you can also use `fn. with_preemption_mode("allow")` instead